src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const amount = days * 40;
```

Comment:
Also please try to add variables for all values
![image](https://user-images.githubusercontent.com/54936044/226142100-7e192edd-e7b1-4806-b48d-52a4e56d3e0b.png)